pkg/apis/apps.kubermatic/v1/application_definition.go

```diff
@@ -206,6 +206,12 @@ type ApplicationDefinitionSpec struct {
 	// SourceURL holds a link to the official source code mirror or git repository of the application
 	SourceURL string `json:"sourceURL,omitempty"`
 
+	// Logo of the Application as a base64 encoded svg
+	Logo string `json:"logo,omitempty"`
+
+	// LogoFormat contains logo format of the configured Application. Options are "svg+xml" and "png"
+	LogoFormat string `json:"logoFormat,omitempty"`
```

Comment:
> "svg+xml" and "png"

If these are the only options then add validation, kubebuilder marker/notations, for this field.

Reply:
good point, have added it.